ui/component/or-attribute-input/src/index.ts

```diff
@@ -133,19 +144,59 @@ export function getAttributeValueTemplate(
                 if (inputType === InputType.JSON && value !== null && typeof(value) !== "string") {
                     value = JSON.stringify(value, null, 2);
                 }
-                return Util.getAttributeValue(attribute, attributeDescriptor) ? Util.getAttributeValue(attribute, attributeDescriptor) : value;
+                currentValue = Util.getAttributeValue(attribute, attributeDescriptor) ? Util.getAttributeValue(attribute, attributeDescriptor) : value;
+                return currentValue;
+            };
+
+            // todo: using this fn to put a value into current value so the button knows about it is a bit of a detour but it works for now
+            const setHelperTextChanged = (event: any) => {
+                currentValue = event.path[0].value;
+                helperText = i18next.t("pressEnterToSend");
             };
 
             const setValue = (value: any) => {
+                iconTrailing = "send-clock";
+                helperText = i18next.t("sending");
```

Comment:
This does not update the helper text for some reason. It does reach this point in the code though.